NorthstarDLL/logging/logging.cpp

```diff
@@ -261,12 +261,18 @@ void StartupLog()
 		spdlog::info("Operating System: {} (Wine)", sysname);
 		spdlog::info("Wine build: {}", wine_get_build_id());
 
+		// STEAM_COMPAT_TOOL_PATHS is a colon separated lists of all compat tool paths used
+		// The first one tends to be the Proton path itself
+		// We extract the basename out of it to get the name used
 		char* compatToolPtr = std::getenv("STEAM_COMPAT_TOOL_PATHS");
 		if (compatToolPtr)
 		{
 			std::string compatToolPath(compatToolPtr);
 
-			spdlog::info("Proton build: {}", compatToolPath.substr(compatToolPath.rfind("/") + 1));
+			auto protonBasenameEnd = compatToolPath.find(":");
```

Comment:
If there is only one compat tool, wont this be `std::string::npos`? This wouldn't break the later code would it?

Reply:
yeah, it would.

Reply:
Should be fixed by c92531ff6a1ade76546589fae16d50e5cc7b630f